clientServiceClient/api.go

```diff
@@ -33,6 +33,16 @@ type ClientGetPublicResponse struct {
 	PublicClient
 }
 
+// ClientBatchPublicRequest is a list of clientIDs to get public information for.
+type ClientBatchPublicRequest struct {
```

Comment:
suggest: `BatchClientInfoRequest` or `BatchClientPublicRequest` , I'm not a fan of naming things in a way that leads to more questions(its clear to you today with public means but it will it be so obvious in the future for you and others?(not withstanding the helpful doc comment))

Reply:
👍 went for public to differentiate a standard Info request from a public request, but maybe even further: `BatchClientPublicInfoRequest` is the way to go.